publishable/assets/css/rtl.css

```diff
@@ -276,6 +276,16 @@
     float: right;
 }
 
+[dir="rtl"]  .voyager .navbar .navbar-nav .dropdown-menu, [dir="rtl"] .voyager .navbar.navbar-default .navbar-nav .dropdown-menu {
```

Comment:
The second selector is not necessary here as it's a subset of the first (.navbar vs .navbar.navbar-default) and therefore adds no value.